controllers/getArticles.controllers.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticlesById } = require('../models/getArticles.models')
+
+exports.getArticles = (req, res, next) => {
+
+    const article_id = req.params.article_id
+    // console.log(article_id)
+    
+    fetchArticlesById(article_id)
+        .then((article) => {
+            if (article.length === 0) {
+               return Promise.reject({status: 404, message: 'Article not found'})
```

Comment:
if you're inside the .then block of your controller this says to me that the model was invoked without any error. I would expect there to be an error and for it to be rejected in the models, then the model will either enter .then will a successful request to the db, or the catch if there isn't